api/package.json

```diff
@@ -18,6 +18,7 @@
     "express-session": "^1.17.1",
     "mongodb": "^6.3.0",
     "mongodb-client-encryption": "^6.0.0",
```

Comment:
We should be able to remove `mongodb` and `mongodb-client-encryption` (not entirely sure about the second one).